Modules/quote/View/Quotelist/editnewuserAction.php

```diff
@@ -25,7 +25,18 @@
         }
         ?>
         <h3><?php echo QuoteTranslator::EditQuote($lang) ?></h3>
+        <div>
+            <a class="btn btn-default" href="clclientedit/<?php echo $id_space ?>">
+                <?php echo ClientsTranslator::NewClient($lang) ?>
+            </a>
+        </div>
         <?php echo $formHtml ?>
+        <script type="module">
+            import {DynamicForms} from '/externals/pfm/dynamics/dynamicForms.js';
+            let dynamicForms = new DynamicForms();
+            dynamicForms.dynamicFields("id_client", "id_pricing", `clientspricingsgetclientpricing`);
+            dynamicForms.dynamicFields("id_client", "address", `clientslistgetaddress`);
```

Comment:
you will add 2 onchange triggers on id_client (or second one will overwrite the first)
you should: 1 register the "triggers" and on change loop over triggers